### ISSUE TITLE
Document language missing - The language of the document is not identified.

### ISSUE THREAD

**Issue by mgifford** [RocketChat/Rocket.Chat#10378]
### Description:

In the web app the document language is missing.  For WCAG 2.0 AA, the language of the document needs to be identified.

I'm not sure what version I'm using. I'm just a user.

### Steps to Reproduce:

1. go to web interface
2. run the wave toolbar http://wave.webaim.org/extension/
3. look at all the red errors

### Expected behavior:

There should be a title. 

### Actual behavior:

There is no title.

There also doesn't seem to be a generated by metatag which would allow me to know what version of the system I am using. 

<img width="870" alt="screenshot of webaim results for page." src="https://user-images.githubusercontent.com/116832/38456606-497e4264-3a54-11e8-9587-6e3cd917a5a5.png">

I can't tag this for accessibility but it needs to be tied to https://github.com/RocketChat/Rocket.Chat/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+accessibility

**Comment by hugocostadev** [comment]
Hi there, thanks for the contribution! 🚀 💯 

Sorry, this error does not happen on the latest and maintained releases.

[Rocket.Chat](https://rocket.chat/)  provides support for, at least, 3 months for each release and 6 months to the latest releases of each major version

I would suggest you update your Rocket.Chat version to the latest stable version, which you can find in [Releases Page](https://github.com/RocketChat/Rocket.Chat/releases) and, check it [HERE](https://docs.rocket.chat/resources/get-support/enterprise-support#rocket.chat-versions) the table with versions and End of Life dates. 

---

**Questions? Help needed? Feature Requests?** 

- Join our [Open Server](https://open.rocket.chat/) in the #support channel and feel free to raise a question
- Join our [Community Forum](https://forums.rocket.chat/) and search/create a post there
- Feature Request: Open an issue in [Feature Request](https://github.com/RocketChat/feature-requests) repository
